src/chains/ethereum/ethereum/src/provider-events.ts

```diff
@@ -37,27 +37,25 @@ function normalizeEvent(event: InterpreterStep) {
 
   return {
     account: {
-      nonce: Quantity.from(account.nonce.toArrayLike(Buffer)).toBigInt(),
-      balance: Quantity.from(account.balance.toArrayLike(Buffer)).toBigInt(),
+      nonce: account.nonce,
+      balance: account.balance,
       stateRoot,
       codeHash
     },
     address,
     codeAddress,
     depth: BigInt(event.depth),
-    gasLeft: Quantity.from(event.gasLeft.toArrayLike(Buffer)).toBigInt(),
-    gasRefund: Quantity.from(event.gasRefund.toArrayLike(Buffer)).toBigInt(),
+    gasLeft: event.gasLeft,
+    gasRefund: event.gasRefund,
     memory,
-    memoryWordCount: Quantity.from(
-      event.memoryWordCount.toArrayLike(Buffer)
-    ).toBigInt(),
+    memoryWordCount: event.memoryWordCount,
     opcode: {
       name: opcode.name,
       fee: opcode.fee
     },
     pc: BigInt(event.pc),
-    returnStack: event.returnStack.map(rs => rs.toArrayLike(Buffer)),
-    stack: event.stack.map(s => s.toArrayLike(Buffer))
+    returnStack: event.returnStack.map(r => r),
+    stack: event.stack.map(s => s)
```

Comment:
This one makes me nervous because gas estimation is really complicated and finicky and we don't have any tests for it yet :/ I don't know if we were doing this to prevent passing those arrays by reference